supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -784,6 +794,16 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 *See also*:
 
+[[restore]]
+==== image:images/yes.png[yes] restore (verb)
+*Description*: To _restore_ is to correct or replace data that was lost or corrupted after a problem occurred in the cluster.
```

Comment:
This definition of "restore" is cluster-specific. Update to add which product or situation this definition applies in.

Reply:
Updated to "In Red Hat OpenShift"